gozw.go

```diff
@@ -300,7 +300,24 @@ func (c *Client) FactoryReset() error {
 
 func (c *Client) AddNode() (*Node, error) {
 	prog := make(chan PairingProgressUpdate, 1)
-	ctx, _ := context.WithTimeout(context.Background(), 3*time.Minute) // Times out after 3 minutes
+	ctx, _ := context.WithTimeout(context.Background(), 1*time.Minute) // Times out after 1 minute
```

Comment:
What happens if pairing takes longer than 1 minute? At 5 seconds per CC interview and 20 CCs for one of our devices we would timeout before we've gotten to interview all devices in the worst case.

Reply:
Makes sense. I'll increase this back to 3 minutes for the time being.